pipeline-templates/prepare-integration-test-steps.yml

```diff
@@ -17,10 +17,17 @@ steps:
       command: custom
       custom: tool
       arguments: install dotnet-stryker -g --version $(IntegrationTestVersion) --add-source $(MygetFeedUri)
+
   - task: DotNetCoreCLI@2
     displayName: 'Install integrationtest dotnet tool'
     condition: eq(variables['System.PullRequest.IsFork'], 'False')
     inputs:
       command: custom
       custom: tool
       arguments: install dotnet-stryker -g --version $(IntegrationTestVersion) --add-source $(AzureArtifactFeedUri)
+
+  - task: CacheBeta@0
```

Comment:
Can the pipeline template not be used here?

Reply:
Sorry, missed a task earlier. Should be resolved in commit e45136f